packages/components/src/SidePanel/SidePanel.component.js

```diff
@@ -62,12 +62,19 @@ function SidePanel({ id, selected, onSelect, actions, docked, onToggleDock, t, r
 					/>
 				</li>
 				{actions.map(action => {
-					const isSelected = isActionSelected(action);
 					const a11y = {};
+					const extra = {};
+					const isSelected = isActionSelected(action);
+					const actionId =
+						action.id ||
+						action.label
+							.toLowerCase()
+							.split(' ')
```

Comment:
a regexp could be used here but i guess it is a lot more readable the way you did it